data/json/snippets/tutorial.json

```diff
@@ -185,7 +185,7 @@
       },
       {
         "id": "LESSON_CRAFTING_FOOD",
-        "text": "Looks like you're hungry!  The hunger status is indicated on the sidebar.  Let's make a <color_light_cyan>peanut butter sandwich</color> from <color_light_cyan>peanut butter</color> and <color_light_cyan>crackers</color> on the counter next to the wall.\n\nTo make food, you need to use the <color_light_cyan>Crafting menu</color>.  Press <keybind:craft> to open the menu.  Press <keybind:NEXT_TAB> and <keybind:PREV_TAB> to toggle crafting categories and select the <color_light_cyan>FOOD</color> category.  From there, select peanut butter sandwich and press <keybind:CONFIRM> to start crafting.\n\nNote that you don't need to stand directly near the counter to be able to craft.  The <color_light_cyan>crafting radius</color> is 6 tiles, so while you're standing no farther than 6 tiles from any crafting components and tools, your character is able to still get them in order to use them.\n\nTo continue the tutorial, please craft the peanut butter sandwich and then eat it by pressing <keybind:eat>."
+        "text": "Looks like you're hungry!  The hunger status is indicated on the sidebar.  Let's make a <color_light_cyan>peanut butter sandwich</color> from <color_light_cyan>peanut butter</color> and <color_light_cyan>crackers</color> on the counter next to the wall.\n\nTo make food, you need to use the <color_light_cyan>Crafting</color> menu.  Press <keybind:craft> to open the menu.  Press <keybind:NEXT_TAB> and <keybind:PREV_TAB> to toggle crafting categories and select the <color_light_cyan>FOOD</color> category.  From there, select peanut butter sandwich and press <keybind:CONFIRM> to start crafting.\n\nNote that you don't need to stand directly near the counter to be able to craft.  The <color_light_cyan>crafting radius</color> is 6 tiles, so while you're standing no farther than 6 tiles from any crafting components and tools, your character is able to still get them in order to use them.\n\nTo continue the tutorial, please craft the peanut butter sandwich and then eat it by pressing <keybind:eat>."
```

Comment:
Actually, it has no name in-game (the keybinding for it is called "Craft Items") so my bad, but I still think it's better that way.